addons/calendar/views/mail_activity_views.xml

```diff
@@ -7,26 +7,26 @@
         <field name="inherit_id" ref="mail.mail_activity_view_form_popup"/>
         <field name="arch" type="xml">
             <xpath expr="//field[@name='date_deadline']" position="attributes">
-                  <attribute name="attrs">{'invisible': [('activity_category', '=', 'meeting')]}</attribute>
+                  <attribute name="attrs">{'invisible': ['|', ('activity_category', '=', 'meeting'), ('activity_category', '=', 'scheduled_phonecall')]}</attribute>
             </xpath>
             <xpath expr="//field[@name='user_id']" position="attributes">
-                  <attribute name="attrs">{'invisible': [('activity_category', '=', 'meeting')]}</attribute>
+                  <attribute name="attrs">{'invisible': ['|', ('activity_category', '=', 'meeting'), ('activity_category', '=', 'scheduled_phonecall')]}</attribute>
             </xpath>
             <xpath expr="//button[@name='action_close_dialog']" position="attributes">
-                  <attribute name="attrs">{'invisible': ['|', ('activity_category', '=', 'meeting'), ('id', '!=', False)]}</attribute>
+                  <attribute name="attrs">{'invisible': ['|', ('activity_category', '=', 'meeting'), ('activity_category', '=', 'scheduled_phonecall')]}</attribute>
             </xpath>
             <xpath expr="//button[@name='action_done']" position="attributes">
-                  <attribute name="attrs">{'invisible': ['|', ('activity_category', '=', 'meeting'), ('chaining_type', '=', 'trigger')]}</attribute>
+                  <attribute name="attrs">{'invisible': ['|', ('activity_category', '=', 'meeting'), ('activity_category', '=', 'scheduled_phonecall')]}</attribute>
             </xpath>
             <xpath expr="//button[@special='cancel']" position="attributes">
-                  <attribute name="attrs">{'invisible': [('activity_category', '=', 'meeting')]}</attribute>
+                  <attribute name="attrs">{'invisible': ['|', ('activity_category', '=', 'meeting'), ('activity_category', '=', 'scheduled_phonecall')]}</attribute>
             </xpath>
             <xpath expr="//field[@name='note']" position="attributes">
-                  <attribute name="attrs">{'invisible': [('activity_category', '=', 'meeting')]}</attribute>
+                  <attribute name="attrs">{'invisible': ['|', ('activity_category', '=', 'meeting'), ('activity_category', '=', 'scheduled_phonecall')]}</attribute>
             </xpath>
             <xpath expr="//button[@name='action_close_dialog']" position="before">
                 <button string="Open Calendar"
-                    attrs="{'invisible': [('activity_category', '!=', 'meeting')]}"
+                    attrs="{'invisible': [('activity_category', '!=', 'meeting'), ('activity_category', '!=', 'scheduled_phonecall')]}"
```

Comment:
Could replace all these by "in" and "not in" syntax to save some space and ease readability.